schedule/yast/maintenance/lvm_thin_provisioning_sle15sp3_dev.yaml

```diff
@@ -0,0 +1,20 @@
+---
+name: lvm_thin_provisioning_dev
+vars:
+  DUD_ADDONS: sdk
+  YUI_REST_API: 1
+schedule:
+  suggested_partitioning:
+    - installation/partitioning/new_partitioning_gpt
+  system_role:
+    - installation/system_role/accept_selected_role_SLES_with_GNOME
+  booting:
+    - installation/installation_settings/validate_default_target
```

Comment:
let's add the hook `default_systemd_target` in `default_sle15sp3` in the same way that was set for `default.yaml`. It avoids the repetition of the disable_boot_menu_timeout, it is connected with boot you are right on that, but not with any screen...that is why I put separate until we have better idea.
Please, apply to the other `booting` keys in the PR.